R/system-requirements.R

```diff
@@ -116,11 +116,12 @@ system_requirements_internal <- function(os, os_release, root, package, execute,
     callback <- function(x, ...) invisible()
   }
 
+  if (all(sudo, length(commands))) {
```

Comment:
I think this condition would written more clearly like this
```suggestion
  if (sudo && length(commands) > 0) {
```